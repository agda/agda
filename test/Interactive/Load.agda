module Load where

data Bool : Set where
  tt : Bool
  ff : Bool
