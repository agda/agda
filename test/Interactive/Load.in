:load test/Interactive/Load.agda
:quit
