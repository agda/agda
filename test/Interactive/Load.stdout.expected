Checking Load
test/Interactive/Load.agda
